src/client/Client.cpp

```diff
@@ -163,6 +173,10 @@ sf::Uint32 Client::getId() const {
 
 void Client::handleEvents(sf::Event& event) {
     if (event.type == sf::Event::KeyPressed) {
+        if (sf::Keyboard::isKeyPressed(mKeys.group)) {
+            mGroupable ^= true;
```

Comment:
why do we need an XOR here?

Reply:
xor just toggles the current value of the boolean.

1 ^ 1 = 0
0 ^ 1 = 1

Reply:
nice